### ISSUE TITLE
Ledger: Mineral doesn't have name

### ISSUE THREAD

**Issue by Geobert** [EDDiscovery/EDDiscovery#690]
5.2.6
In the ledger, in Notes column, I can see "$osmium_name;" "$praseodymium_name;" "$lepidolite_name;" and others

**Comment by klightspeed** [comment]
These names are directly from the journal.

Is this in the 2.2.03 beta, or in live?

Sounds like Frontier changed the names of some commodities from their standard names to `$symbolic_name;`

**Comment by Geobert** [comment]
It's in live

**Comment by klightspeed** [comment]
```
{ "timestamp":"2017-01-09T10:36:24Z", "event":"MarketBuy", "Type":"lepidolite", "Count":1, "BuyPrice":477, "TotalCost":477 }
{ "timestamp":"2017-01-09T11:02:44Z", "event":"EjectCargo", "Type":"lepidolite", "Count":1, "Abandoned":false }
{ "timestamp":"2017-01-09T11:03:14Z", "event":"CollectCargo", "Type":"lepidolite", "Stolen":false }
{ "timestamp":"2017-01-09T11:13:24Z", "event":"EjectCargo", "Type":"lepidolite", "Count":1, "Abandoned":false }
{ "timestamp":"2017-01-09T11:13:49Z", "event":"CollectCargo", "Type":"lepidolite", "Stolen":false }
```

![image](https://cloud.githubusercontent.com/assets/147116/21764977/a890db02-d6b1-11e6-8ea0-2874722c6050.png)

This suggests that at some point between E:D 2.2 and now, that Lepidolite might have been named `$lepidolite_name;`

If you match the ledger against the journal, what are the journal lines you're seeing?

**Comment by Geobert** [comment]
In the history everything is displayed correctly

**Comment by phroggster** [comment]
A fix has been pushed and will be included in the upcoming 5.2.9 release. However, you may notice that your commodity counts are adjusted, as MiningRefined, MissionAccepted, and MissionCompleted events were not accurately accounted for previously.

**Comment by finwen** [comment]
Should be solved in 5.2.9
